homeassistant/components/synology_dsm/__init__.py

```diff
@@ -690,13 +705,15 @@ def available(self) -> bool:
         return bool(self._api.storage)
```

Comment:
Let's add `# type: ignore` here and it all similar places. I was thinking about that and `# type: ignore` has advantage that when underlying library will become typed, mypy will tell this ignore is not needed. With `bool` it won't tell that.